### DISCUSSION TITLE
Github.com address is resolved. Services and websites still not accessible

### DISCUSSION THREAD

**Discussion by Madtricksystem** [community/community#23931] [New to GitHub]
<p>Hello.<br>
I have freshly installed my system.<br>
Windows 10 Pro</p>
<p>Now I am setting up the development environment for the first time.<br>
So far I am now using the default browser Edge.</p>
<p>Here I get the website <a href="https://Github.com" rel="noopener nofollow ugc">https://Github.com</a> almost not displayed. It then comes the following error message:<br>
ERR_CONNECTION_REFUSED</p>
<p>As a test I have installed the Opera browser. There then the same problem with the same error message.</p>
<p>As a further test I installed Firefox. As in the previous system. Here the connections to the website work perfectly. Without error messages etc.<br>
I don’t remember ever having this problem on the old system. Okay, … I only had Firefox in use there. This should no longer be the case.</p>
<p>To track down the problem I first checked the network.<br>
Actually also standard. Modem, router, switch, VLANs, computer.<br>
No proxies. The settings for proxies and network in the system are also correct.<br>
The Windows firewall does not block the browsers either. Opera and Edge connections are allowed.</p>
<p>What can be the problem here.</p>
<p>The Internet search does not give me any clues as to what the problem could be. Maybe I am using the wrong search terms. But what exactly should I search for, if I don’t know exactly what the problem could be.<br>
Of course I could continue to use Firefox as before. But I actually want to get away from it for this system and stay close to the installation standards. So with the Edge browser.</p>
<p>The system is backed up as an image after installation with basic settings like network, name, cloud, etc. and can be reset to that state at any time.</p>
<p>I appreciate any help in finding the problem and would appreciate your comments.</p>
<p>Kind regards<br>
Dirk</p>

**Comment by Madtricksystem** [comment, marked as answer]
<p>The evil is found.<br>
It is due to my firewall pfSense. The ad blocker pfBlockerNG has filtered out the IPs of <a href="http://Github.com" rel="noopener nofollow ugc">Github.com</a>. Why the IPs of Github is in one of the hourly automatically downloaded exclusion lists I can not say unfortunately.<br>
Maybe I can find out which provider of the lists has put the Github IPs on the index as advertisements.</p>
<p>That’s the end of the question here.<br>
Should I delete the post? Better maybe to leave it so someone else can possibly benefit from it.</p>

**Comment by canuckjacq** [comment]
<p>I’m glad you found the cause! I’m always on the side of leaving your work for others to find :slight_smile:</p>
<p>You never know how many people will find this and you’ll have saved them hours!</p>
